doc/source/contributing.rst

```diff
@@ -75,7 +75,7 @@ Some great resources for learning Git:
 
     * the `GitHub help pages <http://help.github.com/>`_.
     * the `NumPy's documentation <http://docs.scipy.org/doc/numpy/dev/index.html>`_.
-    * Matthew Brett's `Pydagogue <http://matthew-brett.github.com/pydagogue/>`_.
+    * Matthew Brett's `The curious coder’s guide to git <https://matthew-brett.github.io/curious-git/>`_.
```

Comment:
I think the replacement link should be https://matthew-brett.github.io/pydagogue/ ; the Curious Git guide is also linked from the pydagogue page.